app/containers/ReviewForm/ReviewForm.tsx

```diff
@@ -202,6 +202,7 @@ class ReviewForm extends React.Component<ReviewFormProps, ReviewFormState> {
             <Col md={6}>
               <ReactStars
                 count={5}
+                color2={'#7ed964'}
```

Comment:
I'd like to avoid having this hex color repeated in multiple places so it's easier to change next time.  Could you either a) add a constant in app/utils/constants.js b) wrap the ReactStars with a component that specifies this color for us.

Reply:
Yeah I had the same thoughts, but I just wasn't sure where to place it. It should probably be in the constants.js file like you said.